src/databricks/sql/thrift_backend.py

```diff
@@ -614,7 +613,10 @@ def _create_arrow_table(self, t_row_set, lz4_compressed, schema_bytes, descripti
                 num_rows,
             ) = convert_column_based_set_to_arrow_table(t_row_set.columns, description)
         elif t_row_set.arrowBatches is not None:
-            (arrow_table, num_rows,) = convert_arrow_based_set_to_arrow_table(
+            (
+                arrow_table,
+                num_rows,
```

Comment:
Python doesn't require these parentheses and they affect readability. Why not?

```python
arrow_table, num_rows = convert_arrow_based_set_to_arrow_table( ... )
```

Reply:
this was just the result of running black on the whole file. Not sure, but if we standardize to a linter, we should use it on everything